src/shared/components/mat-dialog/mat-dialog.component.html

```diff
@@ -0,0 +1,33 @@
+<div class="container">
+  <div mat-dialog-title class="header">
+    <h1 *ngIf="data.type == 'add'">Create post</h1>
```

Comment:
use ngSwitch or switch/case in method in .ts

Reply:
aproove